src/components/ArticlesList.jsx

```diff
@@ -29,12 +29,8 @@ export default function ArticlesList() {
           <div id="article--deck">
             {articles.map((article) => {
               return (
-                <div key={article.article_id} id="article--tile">
+                <div key={article.article_id} class="article--tile">
```

Comment:
You have a warning in your console to remove class - change to className please 😄